### PR TITLE
Adding docs about MockSerializationResult.

### PR DESCRIPTION
**Pull request by renanigt**
In the last days, some people asked on Google Groups about how to test a Controller method that just serialize an object.

So, I wrote this little doc about `MockSerializationResult`.

### CODE REVIEW

File: vraptor-site/content/pt/docs/testando-componentes-e-controllers.html [hunk 1]
```diff
@@ -93,6 +93,42 @@
 
 Note que qualquer chamada do tipo `result.use(...)` será ignorada.
 
+##MockSerializationResult
+
+Muito semelhante ao `MockResult` com a diferença que podemos obter o conteúdo serializado no `Result`.
```
Comment (Turini) [inline vraptor-site/content/pt/docs/testando-componentes-e-controllers.html]: I'd just improve the context here, using something like: *"De forma bastante
semelhante, você pode utilizar o `MockSerializationResult` sempre que 
quiser inspecionar o conteúdo serializado no `Result`"* What do you think?

File: vraptor-site/content/en/docs/testing-components-and-controllers.html [hunk 2]
```diff
@@ -94,6 +94,42 @@
 
 Note that any call of the type `result.use(...)` will be ignored.
 
+##MockSerializationResult
+
+Very similar to `MockResult` with a difference that we can get the serialized content from `Result`.
```
Comment (Turini) [inline vraptor-site/content/en/docs/testing-components-and-controllers.html]: the same here: *"in a similar way, you can use the `MockSerializationResult` 
anytime you want to inspect the serialized content in `Result`"* (or something)

**Comment by Turini** [comment]
Besides the minor I've mentioned, it looks good to :rocket:.
It's nice to see you improving VRaptor docs again, @renanigt :)

**Comment by renanigt** [comment]
Done.

**Comment by renanigt** [comment]
>It's nice to see you improving VRaptor docs again, @renanigt :)

Thanks @Turini . I like to help the VRaptor community, even with my little knowledge. I learn a lot with this and you contributors. :smiley:

**Comment by Turini** [comment]
merged! thanks again @renanigt

**Pull request merged**